README.md

```diff
@@ -92,6 +92,21 @@ A vDU profile can be applied to the image before baking with
 make vdu
 ```
 
+### Reissue external API certificate to match the original cluster FQDN
+When using this, the kubeconfig of the original cluster before the ingrade will be working to access the cluster after the ingrade.
+
+#### How it works
+- A new certificate is created and signed by the loadbalancer CA with a DNS alternative name of `dummy.recert.name`
+- This certificate is included in the relocation configuration
+- When `recert` runs, it will change that dummy name to the one of the running cluster
```

Comment:
```suggestion
- When `recert` runs, we will use it to change that dummy name to the one of the running cluster
```

Reply:
Do we really need this?
Why not create the cert with the right CN? we know what the CN should be at this point

Reply:
Because we want to be as similar as possible to what we will do in "prod", and this is something we'll have to do